README.md

```diff
@@ -152,22 +152,23 @@ This snippet will ensure your new service will skip deployment to production, bu
 
 ## Test
 
-There are couple tests for SaaS Herder. To run them, you can use included `tests/Dockerfile.test`
+SaaS Herder tests are run in an isolated container, but talks to the host docker
+daemon to talk to the oc cluster.
 
-```
-docker build -t saasherder-test -f tests/Dockerfile.test .
-```
+NOTE: You might have to additionally enable
+[Insecure Registries](https://wiki.archlinux.org/index.php/Docker#Insecure_registries) on
```

Comment:
Mind adding a not that the link is only for Arch Linux? (Fedora and Centos can be easily configured through sysconfig)